src/models/Community.ts

```diff
@@ -0,0 +1,61 @@
+import { Schema, model, models } from "mongoose";
+import type { Types, Model, PopulatedDoc } from "mongoose";
+import type { InterfaceOrganization } from "./Organization";
+
+/**
+ * This is an interface that represents a database(MongoDB) document for Community.
+ */
+export interface InterfaceCommunity {
+  _id: Types.ObjectId;
+  name: string;
+  image: string | undefined;
```

Comment:
Rename as stated above.

Reply:
![image](https://github.com/PalisadoesFoundation/talawa-api/assets/97614113/fcccb342-52e7-4802-a19f-f95f34dd9a6e)

Changed the `image` to `logoUrl`

Reply:
@EshaanAgg I've made changes according to your review. If you are satisfied, could you please resolve these conversations?